_sass/integrations.scss

```
@@ -75,6 +76,11 @@
     .home-integrations-subtext {
       margin: auto 10vw;
       color: var(--color-secondary-medium);
+      // The .parallel-* wedges are opaque and sit at z-index 2; the sibling
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)